application/src/main/java/org/togetherjava/tjbot/commands/Features.java

```diff
@@ -63,6 +63,7 @@ private Features() {
      */
     public static Collection<Feature> createFeatures(JDA jda, Database database, Config config) {
         TagSystem tagSystem = new TagSystem(database);
+        BookmarksSystem bookmarksSystem = new BookmarksSystem(config, database);
```

Comment:
prefer singular, `BookmarkSystem`, reads a bit better.

actually, the same is true for all classes u introduced here. id rename all of them and remove the plural

Reply:
Are you sure? When managing bookmarks you are pretty much always managing multiple bookmarks. You are listing your bookmark**s**, You are removing bookmark**s**, You are adding a new bookmark to your bookmark**s**

Reply:
i thought the command is called `Bookmark`. i still think it reads a bit odd for some of the classes, like `BookmarksSystem`. Its also `TagSystem` and not `TagsSystem`. You also say "computer system" and not "computers system". it just sounds odd.

but for some of ur classes it makes sense. i guess its okay for consistency...

Reply:
I do agree that it is not perfect but the other option does also not sound right for me :/